lib/src/mixins.dart

```diff
@@ -21,17 +28,18 @@ mixin LongPressDetection on StreamDeckPluginAction {
     super.keyUp(event);
     final downTime = _downTime;
 
-    if (downTime != null &&
-        DateTime.now().millisecondsSinceEpoch - downTime > 1000) {
-      longPress();
-    } else {
-      shortPress();
+    if (downTime != null) {
+      final duration = DateTime.now().millisecondsSinceEpoch - downTime;
+      if (duration < 1000) {
```

Comment:
I think we could slightly simplify this to just `if (_longPressTimer != null)`, since if the long press has already fired it'll have been reset to `null` and otherwise we want to `shortPress`.

I don't think there are any cases where the results wouldn't be identical, but it does avoid having the duration coded into two places (which could be a constant, but I think avoiding the calculation at all is cleaner).

Reply:
👍